### PR TITLE
Add cross browser testing docs

### PR DESCRIPTION
**Pull request by cadeParade**
Add page to docs for cross browser testing.

image

image 4

### CODE REVIEW

File: docs/learn/cross-browser-visual-testing.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+
+## Enabling and disabling browsers
+Browsers can be enabled or disabled for any individual project at any time. 
+
+![image of settings page](/images/docs/cross-browser-docs-settings.png)
+
+By default, Firefox is enabled for each new project in Percy. To enable a browser, simply go to Project Settings and click to enable your desired browser. A new baseline will be established when a build is run after a new browser is added to a project. This means that the snapshots in the first build for that browser won’t have a baseline comparison.
+
+You can also disable browsers within Project Settings. Note that if you disable a browser and choose to re-enable it in the future, we will update the browser to Percy’s newest version and the subsequent build will create a new baseline for that browser.
```
Comment (maprules1000) [inline docs/learn/cross-browser-visual-testing.md]: I'd call out the note bit on a new line and make it bold. That way it'll be called out as much as it is in the current project browser settings.

Comment (maprules1000) [inline docs/learn/cross-browser-visual-testing.md]: I'd call out the note bit on a new line and make it bold. That way it'll be called out as much as it is in the current project browser settings.

File: docs/learn/cross-browser-visual-testing.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+Usage & Pricing
```
Comment (maprules1000) [inline docs/learn/cross-browser-visual-testing.md]: It might be a good idea to make `Usage & Pricing` a heading with `##`

Comment (cadeParade) [reply docs/learn/cross-browser-visual-testing.md]: Excellent catch.

**Pull request merged**